app/javascript/packages/address-search/components/in-person-locations.tsx

```diff
@@ -30,6 +31,7 @@ function InPersonLocations({
 
   return (
     <>
+      {ResultsSectionHeading && <ResultsSectionHeading />}
```

Comment:
Is this condition ever `false`? Should it be referring to the lower-case `resultsSectionHeading` ?

```suggestion
      {resultsSectionHeading && <ResultsSectionHeading />}
```

Reply:
Or do we always want it rendered? It's not really clear when we don't want it to be rendered.

```suggestion
      <ResultsSectionHeading />
```

Reply:
The `resultsSectionHeading` is an optional prop that's coming in as a component to be rendered on the tippy top of the results section. I added the `&&` just following the pattern of HeaderComponent in the same file https://github.com/18F/identity-idp/blob/main/app/javascript/packages/address-search/components/in-person-locations.tsx#L40 . It's not coming in, we do not want to render it. In fact, since the use case for this prop isn't merged in yet this PR should ensure we do not see the `resultsSectionHeading` by default.

The uppercase was because I thinking that components would be uppercase by convention but I'm happy to leave it lowercase.

Reply:
Oh, I missed that the prop was being _renamed_ from `resultsSectionHeading` to `ResultsSectionHeading`. That makes sense.

But just to clarify, this is not currently being used? That will come in some future work I'm guessing?

Reply:
@aduth correct! We're setting the stage to use this prop only in the help center. That will come as a later separate PR in the identity-site repo. Feel free to see more details in the ticket https://cm-jira.usa.gov/browse/LG-11857